packages/wonder-blocks-dropdown/src/components/combobox.tsx

```diff
@@ -132,6 +133,14 @@ type Props = {
      */
     // TODO(WB-1740): Add support to `inline` and `both` values.
     autoComplete?: "none" | "list" | undefined;
+
+    /**
+     * The icon (wrapped by a button) that allows opening the listbox widget
```

Comment:
I think the prop docs need to be updated for `startIcon`!

Reply:
oh oops, changing it! (I copy pasted from the impl. spec so I'm going to update it there as well.

Reply:
Fixed! btw I also modified the new story to include both the default and disabled state.

I'm thinking about creating an All Variants story to capture all the Combobox styles, but I'd do it in a separate PR. wdyt?